swift/Tests/Blockchains/HarmonyTests.swift

```diff
@@ -30,4 +31,178 @@ class HarmonyTests: XCTestCase {
         XCTAssertEqual(output.r.hexString, "325aed6caa01a5235b7a508c8ab67f0c43946b05a1ea6a3e0628de4033fe372d")
         XCTAssertEqual(output.s.hexString, "6c19085d3376c30f6dc47cec795991cd37d6d0ebddfa633b0a8f494bc19cd01b")
     }
-}
+
+    func testStakingCreateValidator() {
+        let description = TW_Harmony_Proto_Description.with {
```

Comment:
```suggestion
        let description = HarmonyDescription.with {
```
we have short version typealias, don't use `TW_Harmon_Proto_`

Reply:
TW_Harmony_Proto_Description is auto generated. How can I make it generate the type alias like HarmonyDescription? It is same for other photo types TW_Harmony_Proto_...